delivery/src/main/java/com/wfsample/delivery/DeliveryController.java

```diff
@@ -15,22 +19,34 @@
  *
  * @author Hao Song (songhao@vmware.com).
  */
+@RestController
 public class DeliveryController implements DeliveryApi {
   AtomicInteger tracking = new AtomicInteger(0);
   AtomicInteger dispatch = new AtomicInteger(0);
   AtomicInteger cancel = new AtomicInteger(0);
+  private final Random rand = new Random(System.currentTimeMillis());
+
+  @Value("${request.slow.percentage}")
+  private double percentage;
```

Comment:
can you allow the percentage and latency parameters to be undefined where they can be ignored?

Reply:
Already have a default percentage of 0.0